tests/cli/test_server.py

```diff
@@ -88,6 +89,29 @@ def test_server_start(monkeypatch, macos_platform):
     assert check_output.call_args[1].get("env")
 
 
+def test_server_start_image_versions(monkeypatch, macos_platform):
+    check_call = MagicMock()
+    popen = MagicMock(side_effect=KeyboardInterrupt())
+    check_output = MagicMock()
+    monkeypatch.setattr("subprocess.Popen", popen)
+    monkeypatch.setattr("subprocess.check_call", check_call)
+    monkeypatch.setattr("subprocess.check_output", check_output)
+    version = "master" if len(prefect.__version__.split("+")) > 1 else "latest"
```

Comment:
I think we should parametrize this test with a few mocked values for `prefect.__version__` similar to how we test the docker storage logic